src/openforms/contrib/zgw/clients/zaken.py

```diff
@@ -56,7 +58,7 @@ def set_payment_status(self, zaak: dict, partial: bool = False):
             "betalingsindicatie": "gedeeltelijk" if partial else "geheel",
             "laatsteBetaaldatum": timezone.now().isoformat(),
         }
-        response = self.patch(url=zaak["url"], json=data)
+        response = self.patch(url=zaak["url"], json=data, headers=CRS_HEADERS)
```

Comment:
Not sure about this one, I don't know where `zaak["url"]` points to. If it matches `/zaken/{uuid}` then this is correct

Reply:
yep, it does!